### DISCUSSION TITLE
Installing git on my Mac - error: invalid active developer path

### DISCUSSION THREAD

**Discussion by DataMower** [community/community#22994] [New to GitHub]
<p>Hi all,</p>
<p>I am trying to install git on my Mac from the command line and used the following instructions from <a href="http://git-scm.com" rel="nofollow noopener">git-scm.com</a>:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/230ba381c7a740c6bae6aecde90b7aaab0ee2072.jpeg" data-download-href="/uploads/short-url/501E84ZoMV1cc2cXSdE82wA4Uca.jpg?dl=1" title="Screenshot 2019-10-10 at 19.55.42.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/56166693/181074163-211d5253-f6ba-4c95-96bd-ef95492165bf.jpg" alt="Screenshot 2019-10-10 at 19.55.42.png" data-base62-sha1="501E84ZoMV1cc2cXSdE82wA4Uca" width="416" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/230ba381c7a740c6bae6aecde90b7aaab0ee2072_2_416x500.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/230ba381c7a740c6bae6aecde90b7aaab0ee2072_2_624x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/230ba381c7a740c6bae6aecde90b7aaab0ee2072_2_832x1000.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/230ba381c7a740c6bae6aecde90b7aaab0ee2072_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2019-10-10 at 19.55.42.png</span><span class="informations">1348×1620 367 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Unfortunately, when I type the command git --version, I get the following error:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/a977951221cc0e1ecc1f18e9bfdabb584ecf840d.png" data-download-href="/uploads/short-url/obaTzPKyorDhb52pbV0gHj4sWL3.png?dl=1" title="Screenshot 2019-10-10 at 19.54.48.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/56166693/181074165-ad80d635-6020-492b-892d-f4f7187bd704.png" alt="Screenshot 2019-10-10 at 19.54.48.png" data-base62-sha1="obaTzPKyorDhb52pbV0gHj4sWL3" width="690" height="444" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/a977951221cc0e1ecc1f18e9bfdabb584ecf840d_2_690x444.png, https://aws1.discourse-cdn.com/github/optimized/1X/a977951221cc0e1ecc1f18e9bfdabb584ecf840d_2_1035x666.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/a977951221cc0e1ecc1f18e9bfdabb584ecf840d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/a977951221cc0e1ecc1f18e9bfdabb584ecf840d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2019-10-10 at 19.54.48.png</span><span class="informations">1146×738 180 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Does anybody know what to do, when this type of error occurs? Thanks for your help!</p>

**Comment by pixeldesu** [comment, marked as answer]
<p>You need to install the command line developer tools on Mac OS X. OS X already ships with git and after an update and generally a new machine setup, you need to install these tools for things to work properly.</p>
<p>To do this, simply run this in your Terminal:</p>
<pre><code>xcode-select --install
</code></pre>
<p>and follow the on-screen instructions (which is just a single press of a button, if I remember correctly), and then git should work flawlessly!</p>
